src/ui/buttons/buttons.jsx

```diff
@@ -102,7 +102,9 @@ export function Buttons(props : ButtonsProps) : ElementNode {
     const multiple = fundingSources.length > 1;
 
     if (!fundingSources.length) {
-        throw new Error(`No eligible funding fundingSources found to render buttons:\n\n${ JSON.stringify(fundingEligibility, null, 4) }`);
+        const err = new Error(`No eligible funding fundingSources found to render buttons:\n\n${ JSON.stringify(fundingEligibility, null, 4) }`);
+        err.code = 'validation_error';
```

Comment:
I didn't find any way to use `ERROR_CODES.VALIDATION_ERROR` constant from https://github.com/paypal/paypal-smart-payment-buttons/blob/master/server/config/constants.js#L40, so I used the hard-coded value. Anyone has a suggestion on how to do this?

Reply:
That's a good point. Ideally we should share constants using the sdk-constants repo. Can you send a PR to paypal-sdk-constants that adds an error.js file here: https://github.com/paypal/paypal-sdk-constants/tree/master/src. And then both paypal-checkout-components and smart-payment-buttons can get this from the constants:

```js
export const ERROR_CODE = {
  VALIDATION_ERROR: 'validation_error'
}
```

Reply:
PR is open: https://github.com/paypal/paypal-sdk-constants/pull/59